server/tests/models/services/AtService.test.js

```diff
@@ -0,0 +1,591 @@
+const { sequelize } = require('../../../models');
+const AtService = require('../../../models/services/AtService');
+const randomStringGenerator = require('../../util/random-character-generator');
+const { dbCleaner } = require('../../util/db-cleaner');
+
+afterAll(async () => {
+    await sequelize.close(); // close connection to database
```

Comment:
Why do we need to explicitly close the database? Maybe you could update the comments to explain why?

Reply:
It's generally a safe practice to explicitly close your db connections if no longer needed especially in a testing environment. It will free up space and lessen the chance of any transactional locks.
Also, given that this is a Postgres instance, there's a ~100 max connections pool size so you wouldn't want to unintentionally hit that (although it should never happen) while the build server is attempting to run many test suites that depend on those connections back-to-back

Reply:
Gotcha - maybe remove the comments then!